src/Psalm/Internal/CliUtils.php

```diff
@@ -395,7 +395,7 @@ public static function getPsalmHelpText(): string
 
     --output-format=console
         Changes the output format.
-        Available formats: compact, console, text, emacs, json, pylint, xml, checkstyle, junit, sonarqube, github,
+        Available formats: compact, console, text, emacs, json, pylint, xml, checkstyle, junit, sonarqube, sonarcloud, github,
                            phpstorm, codeclimate
```

Comment:
```suggestion
        Available formats: compact, console, text, emacs, json, pylint, xml, checkstyle, junit, sonarqube, sonarcloud, github,
                           phpstorm, codeclimate
        
        'sonarqube' format is suitable for both SonarQube and SonarCloud
```

Reply:
If I understand it correctly we need to keep both Report Types to keep both `--output-format`. We would prefer to offer both names as a format instead of including an extra hint in the help text.